src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -401,6 +407,19 @@ private async Task StopAndClearConsoleLogsSubscriptionAsync()
         }
     }
 
+    private async Task DownloadLogsAsync()
+    {
+        // Strip control sequences from log lines before combining them into a single string.
+        var formattedLines = _logEntries.GetEntries().Select(entry => entry.RawContent is null ? null : AnsiParser.StripControlSequences(entry.RawContent));
+        var logContent = string.Join(Environment.NewLine, formattedLines);
+
+        var stream = new MemoryStream(Encoding.UTF8.GetBytes(logContent));
+        using var streamReference = new DotNetStreamReference(stream);
+        var fileName = $"{PageViewModel.SelectedResource!.DisplayName}-{DateTime.Now.ToString("yyyyMMddhhmmss", CultureInfo.CurrentCulture)}.txt";
```

Comment:
I don't think there is any reason to use the current culture for this:
```suggestion
        var fileName = $"{PageViewModel.SelectedResource!.DisplayName}-{DateTime.Now.ToString("yyyyMMddhhmmss", CultureInfo.InvariantCulture)}.txt";
```